src/components/Toast/Toast.css

```diff
@@ -0,0 +1,36 @@
+.toast {
+  align-items: center;
+  display: flex;
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  background-color: #08d69f;
+  width: auto;
```

Comment:
what happens when it doesn't recieve any text? or a short text like "Foo"?

I would set a `min-width` of like 200px just to cover those edge cases